### ISSUE TITLE
Use tools for automatic formatting

### ISSUE THREAD

**Issue by m-aguena** [LSSTDESC/CLMM#574]
We should use tools to standardize the code formatting like `black` and `flake8` to keep the code tidy!

**Comment by hsinfan1996** [comment]
Agree, I am using pylint for CLMM because I saw its config in the code.
